apps/passport-server/src/util/instrumentPCDs.ts

```diff
@@ -1,25 +1,34 @@
-import { EdDSAPCDPackage } from "@pcd/eddsa-pcd";
-import { EdDSATicketPCDPackage } from "@pcd/eddsa-ticket-pcd";
-import { EmailPCDPackage } from "@pcd/email-pcd";
 import { PCDPackage } from "@pcd/pcd-types";
-import { RSAImagePCDPackage } from "@pcd/rsa-image-pcd";
-import { RSAPCDPackage } from "@pcd/rsa-pcd";
-import { SemaphoreGroupPCDPackage } from "@pcd/semaphore-group-pcd";
-import { SemaphoreSignaturePCDPackage } from "@pcd/semaphore-signature-pcd";
 import { traced } from "../services/telemetryService";
 import { logger } from "./logger";
 
 export function instrumentPCDs(): void {
   logger(`[INIT] instrumenting PCDs`);
+  /*
+
+Turning this off due to:
+
+[1] [ERROR] POST /issue/verify-ticket
+[1] TypeError: uninstrumentedFunction is not a function
+[1]     at /Users/robknight/Projects/zupass/apps/passport-server/src/util/instrumentPCDs.ts:46:14
+[1]     at Generator.next (<anonymous>)
+[1]     at /Users/robknight/Projects/zupass/apps/passport-server/src/util/instrumentPCDs.ts:8:71
+[1]     at new Promise (<anonymous>)
+[1]     at __awaiter (/Users/robknight/Projects/zupass/apps/passport-server/src/util/instrumentPCDs.ts:4:12)
+[1]     at /Users/robknight/Projects/zupass/apps/passport-server/src/util/instrumentPCDs.ts:42:62
+[1]     at /Users/robknight/Projects/zupass/apps/passport-server/src/services/telemetryService.ts:104:12
+[1]     at Generator.next (<anonymous>)
+[1]     at /Users/robknight/Projects/zupass/apps/passport-server/src/services/telemetryService.ts:8:71
+[1]     at new Promise (<anonymous>)*/
 
   [
-    EdDSAPCDPackage,
+    /* EdDSAPCDPackage,
     EdDSATicketPCDPackage,
     RSAImagePCDPackage,
     RSAPCDPackage,
     EmailPCDPackage,
     SemaphoreSignaturePCDPackage,
-    SemaphoreGroupPCDPackage
+    SemaphoreGroupPCDPackage*/
```

Comment:
This is obviously quite crude but I was getting the above error during testing and didn't want to make proper changes without understanding the purpose behind the code.

Reply:
the error you're seeing is obfuscating the real error which is that somewhere in the code something is trying to call a pcdpackage function that doesn't exist - probably `init`. i got the information i needed from instrumenting this for now, so feel free to leave these packages commented out if you don't want to figure this error out.

Reply:
I've found what is causing this. The cause is in issuance service. Once you remove that one line of code, you'll be able to, and should, revert changes to this file.